src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -202,7 +218,7 @@ private Object createObjectWithIgnoringProblems(String className, Set<String> se
      * @return a string which is obtained by joining package names with a class name.
      */
     private static String joinPackageNamesWithClassName(String className, Set<String> packages) {
-        return packages.stream().filter(name -> name != null)
+        return packages.stream().filter(Objects::nonNull)
```

Comment:
They are equivalent. When writing in the original way, Intellij would raise an inspection error that "It could be replaced by method reference". There are no functional difference but only improving readability.